SearchInRotatedSortedArray.java

```diff
@@ -0,0 +1,92 @@
+//Time Complexity : O(log(n)) Not sure
+//Space Complexity : O(1)
+// Yes, It's run successfully
+// No I don't face any problem.
+
+
+
+public class SearchInRotatedSortedArray {
```

Comment:
Please describe your approach briefly. Also, try to implement `O(logn)` solution.